### DISCUSSION TITLE
xsd schemák

### DISCUSSION THREAD

**Discussion by rrd108** [nav-gov-hu/M2M#51] [Kérdések] [answered]
A doksi hivatkozik a `hibalista.xsd`-re és az általános bizonylat xsd-re.

Ezek hol találhatóa meg?

**Comment by TamasPill** [comment, marked as answer]
A hibalista itt érhető el:
https://github.com/nav-gov-hu/M2M/blob/main/src/SOAP/1.1/external_schema/webnyk/hibalista.xsd

Az általános nyomtatvány xsd pedig pl ennek a dokumentumnak a végén:
https://nav.gov.hu/pfile/file?path=/nyomtatvanyok/letoltesek/nyomtatvanykitolto_programok/nyomtatvany_apeh/keretprogramok/ld_abevjava/Altalanos_nyomtatvanyok_XML_szerkezete_2013.05.13_

**Reply by rrd108** [reply]
Köszi

**Reply by rrd108** [reply]
Viszont ha készítek egy bevallás XML file-t az ÁNYK-val és azt validálom ezzel az xsd-vel akkor eszerint nem valid.

Szeretném beküldés előtt validálni az elkészült xml-t, de így nem tudom.

**Comment by TamasPill** [comment]
Rossz link lett küldve előzőleg.
Ez tartalmazza a helyes xsd-t:
[link](https://nav.gov.hu/pfile/file?path=/nyomtatvanyok/letoltesek/nyomtatvanykitolto_programok/nyomtatvany_apeh/keretprogramok/ld_abevjava/NAV_nyomtatvanyok_XML_szerkezete_2013.05.13_)

**Reply by rrd108** [reply]
Köszi, ezzel már érvényes a validáció
